draft-ietf-quic-http.md

```diff
@@ -586,10 +586,10 @@ using a Stream ID of 0, as in QUIC stream 0 carries a valid HTTP request.  The
 root of the tree cannot be reprioritized.  A PRIORITY frame sent on a request
 stream with the Prioritized Element Type set to any value other than `11` or
 which expresses a dependency on a request with a greater Stream ID than the
-current stream MUST be treated as a stream error of type HTTP_MALFORMED_FRAME.
-Likewise, a PRIORITY frame sent on a control stream with the Prioritized Element
-Type set to `11` MUST be treated as a connection error of type
-HTTP_MALFORMED_FRAME.
+current stream MUST be treated as a connection error of type
+HTTP_MALFORMED_FRAME.  Likewise, a PRIORITY frame sent on a control stream with
+the Prioritized Element Type set to `11` MUST be treated as a connection error
+of type HTTP_MALFORMED_FRAME.
```

Comment:
Since this paragraph is structured as "things that cause a stream error" and "things that cause a connection error," I suspect we could fold everything together if everything is going to be a connection error.  Maybe even something like:

```
The following situations MUST be treated as connection errors of type HTTP_MALFORMED_FRAME:
  - A PRIORITY frame sent on a request stream with the Prioritized Element
    Type set to any value other than `11`
  - A PRIORITY frame which expresses a dependency on a request with a 
    greater Stream ID than the	current stream  
  - A PRIORITY frame sent on a control stream with the Prioritized Element
    Type set to `11`
```